### PR TITLE
Hook up new user help videos

### PR DESCRIPTION
**Pull request by HSalmon**

### CODE REVIEW

File: angular/core/components/group_page/group_help_card/group_help_card.coffee [hunk 1]
```diff
@@ -3,11 +3,14 @@ angular.module('loomioApp').directive 'groupHelpCard', ->
...
   replace: true
-  controller: ($scope, CurrentUser, AppConfig, UserHelpService) ->
+  controller: ($scope, $sce, CurrentUser, AppConfig, UserHelpService) ->
     $scope.showVideo = AppConfig.loadVideos
 
     $scope.helpLink = ->
       UserHelpService.helpLink()
 
+    $scope.helpVideo = ->
+      $sce.trustAsResourceUrl(UserHelpService.helpVideo())
```
Comment (gdpelican) [inline angular/core/components/group_page/group_help_card/group_help_card.coffee]: Wonder if UserHelpService could have a method `helpVideoUrl` which returned the appropriate url, so we didn't have to dup the $sce stuff?

Comment (HSalmon) [inline angular/core/components/group_page/group_help_card/group_help_card.coffee]: Having trouble getting it to work that way...hmmm

Comment (gdpelican) [inline angular/core/components/group_page/group_help_card/group_help_card.coffee]: Does it not like importing $sce into the service? I think I've had issues doing that with stuff like $location in the past. It's not a showstopper by any means.

Comment (HSalmon) [inline angular/core/components/group_page/group_help_card/group_help_card.coffee]: It doesn't seem to be enjoying it, no.

**Comment by gdpelican** [comment]
Perfect! :green_apple:

**Pull request merged**